cdc/sink/codec/schema_registry_test.go

```diff
@@ -0,0 +1,271 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codec
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/jarcoal/httpmock"
+	"github.com/linkedin/goavro/v2"
+	"github.com/pingcap/check"
+	"github.com/pingcap/ticdc/cdc/model"
+)
+
+type AvroSchemaRegistrySuite struct {
+}
+
+var _ = check.SerialSuites(&AvroSchemaRegistrySuite{})
+
+type mockRegistry struct {
+	subjects map[string]*mockRegistrySchema
```

Comment:
What about adding a lock to `mockRegistry` and protect the read/write of `subjects` to avoid data race.